lib/forever/cli.js

```diff
@@ -240,6 +259,8 @@ var getOptions = cli.getOptions = function (file) {
   options.workingDir = options.workingDir || options.sourceDir;
   options.spawnWith  = { cwd: options.workingDir };
 
+  _.extend(options, jsonConfig);
```

Comment:
I'd prefer not to add `lodash` as a dependency for a single function. Lets use this instead: https://github.com/sindresorhus/object-assign